examples/ex25.cpp

```diff
@@ -461,23 +455,21 @@ int main(int argc, char *argv[])
       gmres.SetMaxIter(2000);
       gmres.SetRelTol(1e-5);
       gmres.SetAbsTol(0.0);
-      gmres.SetOperator(*A);
+      gmres.SetOperator(*Ah.As<ComplexSparseMatrix>());
```

Comment:
Maybe use `gmres.SetOperator(*Ah.Ptr());` since `OperatorHandle` could be used as a `ComplexOperator` for PA, as well.